.github/workflows/publish-docs.yml

```diff
@@ -76,22 +90,22 @@ jobs:
           node-version: ${{ env.NODE_VERSION }}
 
       - name: mike deploy main
-        if: "!github.event.workflow_dispatch && contains(github.ref, 'refs/heads/main')"
+        if: contains(github.ref, 'refs/heads/main')
```

Comment:
I'm not sure about this.. maybe it should be only done when it's not a manual dispatch